clients/admin-ui/cypress/e2e/integration-management.cy.ts

```diff
@@ -49,6 +55,13 @@ describe("Integration management for data detection & discovery", () => {
         cy.getByTestId("empty-state").should("not.exist");
       });
 
+      it("should be able to test connections by clicking the button", () => {
+        cy.getByTestId("integration-info-bq_integration").within(() => {
+          cy.getByTestId("test-connection-btn").click();
+          cy.wait("@testConnection");
```

Comment:
do you think it's worth checking for the toast here as well?

Reply:
@gilluminate I tried to do that originally but wasn't able to actually test for it-- there's an onQueryStarted helper method that runs when the test query is run (see `datastore-connection.slice.ts`) and it throws an uncaught error in the Cypress environment.  I wasn't able to figure out how to suppress or stub a response or anything to run more Cypress commands after the test query.

Reply:
I see. That's unfortunate! Ship it!